internal/db/seeds/dev_small.sql

```diff
@@ -1,15 +1,20 @@
 --
 -- Testing seed data, to execute this file during migration set DB_SEED variable to "dev_small".
--- Keep this file idempotent so we can run it over and over again to get
+-- Keep this file idempotent. Always specify primary keys in the range of 1-100.
 --
 BEGIN;
 
-INSERT INTO accounts(account_number, org_id)
+INSERT INTO accounts(id, account_number, org_id)
 VALUES
-    ('13', '000013'),
-    ('15', '000015'),
-    (NULL, '000042'),
-    (NULL, '000077')
+    (1, '13', '000013'),
+    (2, '15', '000015'),
+    (3, NULL, '000042'),
+    (4, NULL, '000077')
+ON CONFLICT DO NOTHING;
+
```

Comment:
Would this work?
```sql
ALTER SEQUENCE accounts_id_seq RESET 5;
```

Reply:
Yes, but we will forget to do this for sure, then its a runtime error.

Reply:
In devel tho, right? and it is easy to discover, tho if we ever hit 100 seeds it would be IMHO quite hard to discover what is going on...